sources/packages/web/src/components/aest/StudentApplications.vue

```diff
@@ -153,6 +153,8 @@ export default {
     const programYear = ref({} as ProgramYearOfApplicationDto);
     const showModal = ref(false);
     const selectedApplicationId = ref(0);
+    const toast = useToastMessage();
+    const TOAST_ERROR_DISPLAY_TIME = 15000;
```

Comment:
Why is the toast time manually set here? I was trying to search in #357 to find any reason. But couldn't get any.

Reply:
to increase the span of TOAST, same is also implemented in the supporting users by other developers